app/main.py

```diff
@@ -0,0 +1,40 @@
+from collections.abc import AsyncIterator
+from contextlib import asynccontextmanager
+import time
+
+from fastapi import FastAPI
+from starlette.requests import Request
+from starlette.responses import Response
+
+from fastapi_cache import FastAPICache
+from fastapi_cache.backends.redis import RedisBackend
+from fastapi_cache.decorator import cache
+
+from redis import asyncio as aioredis
+
+
+@asynccontextmanager
+async def lifespan(_: FastAPI) -> AsyncIterator[None]:
+    redis = aioredis.from_url("redis://redis:6379")
+    FastAPICache.init(RedisBackend(redis), prefix="fastapi-cache")
+    yield
+
+
+app = FastAPI(lifespan=lifespan)
+
+
+@cache()
+async def get_cache():
+    return 1
+
+
+@app.get("/")
+async def index():
+    return dict(status="OK")
+
+
+@app.get("/blocking")
+@cache(namespace="test", expire=10)
```

Comment:
During our call today, we had discussed the possibility of making the `expire` value configurable and also providing a config.yml template. 

Captured here: https://app.zenhub.com/workspaces/vocabulizer-663a95567168690084392fb0/issues/gh/cdcgov/vocabulizer/9